src/Microsoft.VisualStudio.ProjectSystem.Managed/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,78 @@
-const Microsoft.VisualStudio.ProjectSystem.Debug.UIProfilePropertyName.NativeDebugging = "NativeDebugging" -> string
\ No newline at end of file
+Microsoft.VisualStudio.ProjectSystem.ITargetFramework
+Microsoft.VisualStudio.ProjectSystem.ITargetFramework.FrameworkName.get -> System.Runtime.Versioning.FrameworkName
+Microsoft.VisualStudio.ProjectSystem.ITargetFramework.FriendlyName.get -> string
+Microsoft.VisualStudio.ProjectSystem.ITargetFramework.FullName.get -> string
+Microsoft.VisualStudio.ProjectSystem.ITargetFramework.ShortName.get -> string
+Microsoft.VisualStudio.ProjectSystem.ITargetFrameworkProvider
+Microsoft.VisualStudio.ProjectSystem.ITargetFrameworkProvider.GetNearestFramework(Microsoft.VisualStudio.ProjectSystem.ITargetFramework targetFramework, System.Collections.Generic.IEnumerable<Microsoft.VisualStudio.ProjectSystem.ITargetFramework> otherFrameworks) -> Microsoft.VisualStudio.ProjectSystem.ITargetFramework
+Microsoft.VisualStudio.ProjectSystem.ITargetFrameworkProvider.GetTargetFramework(string shortOrFullName) -> Microsoft.VisualStudio.ProjectSystem.ITargetFramework
+Microsoft.VisualStudio.ProjectSystem.VS.Extensibility.IProjectExportProvider
```

Comment:
Lots of types moved to the core layer still have VS in their namespaces. Can we update these and maintain back compat with type forwarding?

Reply:
Never mind. I see type forwarding only applies to matching FQNs.

Reply:
Did we had type forwarding for all the types we moved?

Reply:
See https://github.com/dotnet/project-system/pull/4724/files#diff-2cde465cdf265cc84c5eded3515f7f2f. That's not all moved types. Only the public ones afaict.

Reply:
i guess we only need forward public API types.

Reply:
All types forwarded in this PR are public.